src/greasy.cpp

```diff
@@ -61,7 +61,12 @@ int main(int argc, char *argv[]) {
   }
 
   if (argc != 2) {
+      cout << endl << "Error" << endl;
       cout << "Usage: greasy filename" << endl;
+      cout << "please refer to the documentation https://user.cscs.ch/tools/high_throughput" << endl << endl;
```

Comment:
Errors should go to stderr. People should not that, when they are redirecting. In the normal case, it will look like an error that you print the message twice.